include/sentry.h

```diff
@@ -538,7 +538,8 @@ SENTRY_API void sentry_options_set_http_proxy(
 /*
  * returns the configured http proxy
  */
-SENTRY_API const char *sentry_options_get_http_proxy(sentry_options_t *opts);
+SENTRY_API const char *sentry_options_get_http_proxy(
+    const sentry_options_t *opts);
```

Comment:
Can you make such bug fixes in separate PRs? By looking at the git history, it gets easier to see changes and when writing the changelog we don't miss things (especially when someone else writes it).

Reply:
Sure. I was just mashing together a few oneliners in this PR to avoid the overhead ;-) But yes, that overhead actually serves a very good purpose.